frontend/pages/ManageControlsPage/OSUpdates/components/EndUserOSRequirementPreview/EndUserOSRequirementPreview.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+
+import CustomLink from "components/CustomLink";
+
+import { OSUpdatesSupportedPlatform } from "../../OSUpdates";
+
+import MacOSUpdateScreenshot from "../../../../../../assets/images/macos-updates-preview.png";
+import WindowsUpdateScreenshot from "../../../../../../assets/images/windows-nudge-screenshot.png";
+
+const baseClass = "os-requirement-preview";
+
+interface IOSRequirementDescriptionProps {
```

Comment:
Are this type, `type IOSRequirementImageProps`, and `IEndUserOSRequirementPreviewProps` necessary? Could the functions that use them just take in an `OSUpdatesSupportedPlatform`?

Reply:
100%, great code clean up, this has been in here for months!

Reply:
Glad you agree, did you mean to remove `IOSRequirementDescriptionProps`? It's still in use: 
<img width="389" alt="Screenshot 2024-07-29 at 12 19 34 PM" src="https://github.com/user-attachments/assets/1fc0bd35-6358-46e4-89ba-2d2809929b26">

Reply:
Meant to rename it the component props `IEndUserOSRequirementPreviewProps` since it's being used in 3 places along with the component! Getting that in rn